### ISSUE TITLE
Implement detokenization

### ISSUE THREAD

**Issue by laeubli** [twitter/twitter-korean-text#88]
NLP applications such as [natural language generation](https://en.wikipedia.org/wiki/Natural_language_generation) or [machine translation](https://en.wikipedia.org/wiki/Statistical_machine_translation) produce morpheme-tokenized text if the underlying models are trained on morpheme-tokenized training material which, with agglutinative languages such as Korean, often has a positive impact on performance. The tokenized system output, however, has to be detokenized to accommodate end users.

It would be great if ``TwitterKoreanProcessor`` would implement a method to convert a sequence of tokens back into a string:

```scala
val tokens = List("한국어", "를", "처리", "하는",  "예시", "입니", "다", "ㅋㅋ", "#한국어")
val segment: String = TwitterKoreanProcessor.detokenize(tokens)
// "한국어를 처리하는 예시입니다ㅋㅋ #한국어"
```
Basically speaking, adjacent items in ```tokens``` should be joined together with either an empty string or whitespace, depending on whether or not they belong to the same word unit.

**Comment by nlpenguin** [comment]
Added pull request: https://github.com/twitter/twitter-korean-text/pull/89

@laeubli, could you please take a look and give me a #shipit? (well there's no real #shipit button, so just comment with #shipit.)

**Comment by laeubli** [comment]
Great work, @nlpenguin. Thanks a lot.
